src/styles/blocks/day.scss

```diff
@@ -0,0 +1,26 @@
+.calendar__day {
+  width: $day-size;
+  height: $day-size;
+  border: $border-size solid black;
+  background-color: $grey-color;
```

Comment:
The variable `$grey-color` is used here but is not defined in this file. Ensure that `$grey-color` is defined to avoid compilation errors.